tests/utils/exec/rosacli/cluster_service.go

```diff
@@ -307,6 +308,14 @@ func (c *clusterService) ReflectUpgradeDescription(result bytes.Buffer) (res *Up
 	return res, err
 }
 
+func (c *clusterService) ListUpgrade(clusterID string, flags ...string) (bytes.Buffer, error) {
+	combflags := append([]string{"-c", clusterID}, flags...)
+	describe := c.client.Runner.
+		Cmd("list", "upgrade").
+		CmdFlags(combflags...)
+	return describe.Run()
+}
```

Comment:
Please create a new `Upgrade` service

Reply:
Create Upgrade service with ListUpgrade func.

Reply:
@radtriste Could you take a look? Thanks.